### PR TITLE
Empty State - Style Update and Prop Deprecation

### PR DESCRIPTION
**Pull request by anechol**
## Description

Renames the `scope` prop to `size` and updates styles to match [Figma](https://www.figma.com/design/CC1YmaGKHnsvB28yLY9mEH/Mercury-%E2%86%92-Sage-components?node-id=33742-6953&t=mnFVrLyz83wmYLrF-1).

This PR does not include a gradient.

## Screenshots

|  Before  |  After  |
|--------|--------|
Screenshot 2024-07-30 at 4 35 29 PM|Screenshot 2024-08-02 at 2 03 21 PM

## Testing in `sage-lib`

- Navigate to [Rails](http://localhost:4000/pages/component/empty_state?tab=preview) and [React](http://localhost:4100/?path=/docs/sage-emptystate--default)
- Check that component matches Figma specs.

## Testing in `kajabi-products`

1. (**BREAKING**) - Renames the `scope` prop to `size` and updates styles to match Figma

## Related

[DSS-737](https://kajabi.atlassian.net/browse/DSS-737)

[DSS-737]: https://kajabi.atlassian.net/browse/DSS-737?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]

File: docs/lib/sage_rails/app/sage_components/sage_icon_card.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class SageIconCard < SageComponent
     background_color: [:optional, NilClass, String],
     color: [:optional, NilClass, SageSchemas::STATUSES],
     foreground_color: [:optional, NilClass, String],
+    gradient: [:optional, NilClass, {topColor: String, bottomColor: String}],
```
Comment (ju-Skinner) [inline docs/lib/sage_rails/app/sage_components/sage_icon_card.rb:7]: For my own curiousity, how would this work for left to right gradients?

File: packages/sage-assets/lib/stylesheets/global/_reboot.scss [hunk 2]
```diff
@@ -37,7 +37,7 @@ body {
     "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
   font-feature-settings: "liga" 1, "calt" 1; /* fix for Chrome */
   font-size: 1rem;
-  font-weight: 400;
+  font-weight: 425;
```
Comment (ju-Skinner) [inline packages/sage-assets/lib/stylesheets/global/_reboot.scss:40]: this should not be here.

File: packages/sage-assets/lib/stylesheets/tokens/_font_weight.scss [hunk 3]
```diff
@@ -9,6 +9,13 @@
...
+  regular: 425,
+  medium: 525,
+  semibold: 625,
+  bold: 725
+);
+
+$sage-heading-font-weights: (
   regular: 400,
   medium: 500,
   semibold: 600,
```
Comment (ju-Skinner) [inline packages/sage-assets/lib/stylesheets/tokens/_font_weight.scss:21]: I think you need to rebase.

**Review by ju-Skinner — CHANGES_REQUESTED** [review]

File: docs/lib/sage_rails/app/views/sage_components/_sage_icon_card.html.erb [hunk 4]
```diff
@@ -11,6 +11,7 @@ style = {
     sage-icon-card
     <%= "sage-icon-card--#{color}" if color.present? %>
     <%= "sage-icon-card--round" if component.round %>
+    <%= "sage-icon-card--gradient" if component.gradient %>
```
Comment (ju-Skinner) [inline docs/lib/sage_rails/app/views/sage_components/_sage_icon_card.html.erb:14]: do we want to have this here, even though we aren't using it?

**Review by QuintonJason — APPROVED** [review]
Good to go

**Review by ju-Skinner — APPROVED** [review]

**Pull request merged**